### DISCUSSION TITLE
Powershell as admin?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/terminal#12125] [Q&A] [answered]
I have added a profile that opens powershell as an admin
`{
             "name": "Windows Terminal (elevated)",
            "commandline": "%SystemRoot%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -Command Start-Process -Verb RunAs \"shell:appsFolder\\Microsoft.WindowsTerminal_8wekyb3d8bbwe!App\"",
            "hidden": false,
            "icon": "ms-appx:///Images/Square44x44Logo.targetsize-32.png"
           }`

However this doesn't add it as a tab it opens up a new terminal window. Is there a way to have it as a tab?

**Comment by zadjii-msft** [comment, marked as answer]
Nope. The Terminal doesn't allow mixing of tabs at different elevation levels, because that's a potential attack vector. There's more discussion in #146, #632, #4217, and more specifically https://github.com/microsoft/terminal/issues/632#issuecomment-519375707.

I can't stop you from running a utility like `gsudo`, but you should know that is a potential vulnerability, which is why this feature is not a part of the Terminal by default.
